flocker_bb/builders/flocker_vagrant.py

```diff
@@ -497,8 +499,10 @@ def getBuilders(slavenames):
                       category='flocker',
                       factory=buildTutorialBox(),
                       nextSlave=idleSlave),
-        BuilderConfig(name='flocker/installed-package/fedora-20',
-                      builddir='flocker-installed-package-fedora-20',
+        BuilderConfig(name='flocker/installed-package/' +
```

Comment:
If we are renaming this builder anyway, we should do, `flocker/installed-package/vagrant/{distro}`.